source/reference/api/reverse.txt

```diff
@@ -44,14 +44,33 @@ To use the ``reverse`` endpoint:
   You cannot update these options after the sync starts.
 - ``mongosync`` must be in the ``COMMITTED`` state.
 - Source and destination clusters must be MongoDB 6.0 or later.
-- :ref:`Unique indexes <index-type-unique>` on the original source
-  cluster must be formatted properly. If an upgraded cluster has unique
-  indexes that were created in MongoDB 4.2 or earlier, you must
-  :ref:`resync <resync-replica-member>` all of the nodes in the
-  original source cluster before reversing.
+- :ref:`index-type-unique` requires proper formatting.  To validate
+  that collection indexes use the proper formatting, see
```

Comment:
I think that this bullet should provide more info about only needing to further investigate if the upgraded cluster has unique indexes that were created in mongodb 4.2 or earlier

Reply:
Fixed